tools/syz-showprio/showprio.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+// Tool used to visualize the priorities table.
+import (
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+
+	"github.com/google/syzkaller/pkg/mgrconfig"
+	"github.com/google/syzkaller/pkg/testutils"
+	"github.com/google/syzkaller/prog"
+)
+
+var (
+	flagOS     = flag.String("os", runtime.GOOS, "target os")
+	flagArch   = flag.String("arch", runtime.GOARCH, "target arch")
+	flagEnable = flag.String("enable", "", "comma-separated list of enabled syscalls")
+	flagCorpus = flag.String("corpus", "", "name of the corpus file")
+)
+
+func main() {
+	flag.Parse()
+	target, err := prog.GetTarget(*flagOS, *flagArch)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%v", err)
+		os.Exit(1)
+	}
+	if *flagEnable == "" {
+		fmt.Println("No syscalls enabled")
```

Comment:
Error messages don't start with capital letters.
Also this should go to stderr.